cmd/terramate/cli/run.go

```diff
@@ -454,14 +453,16 @@ func (c *cli) createCloudPreview(runs []runContext) map[string]string {
 		affectedStacksMap[st.Stack.ID] = st.Stack
 	}
 
-	githubEventPath, ok := os.LookupEnv("GITHUB_EVENT_PATH")
-	if !ok {
-		fatal("env var GITHUB_EVENT_PATH not found, not generating previews", nil)
-	}
-
 	prUpdatedAt := time.Now().UTC().Unix()
-	if eventPRUpdatedAt := github.GetEventPRUpdatedAt(githubEventPath); eventPRUpdatedAt != nil {
-		prUpdatedAt = eventPRUpdatedAt.Unix()
+	githubEventPath, ok := os.LookupEnv("GITHUB_EVENT_PATH")
+	if ok {
+		eventPRUpdatedAt := github.GetEventPRUpdatedAt(githubEventPath)
+		if eventPRUpdatedAt != nil {
+			prUpdatedAt = eventPRUpdatedAt.Unix()
+		}
+	} else {
+		printer.Stderr.Warn(
+			sprintf("env var GITHUB_EVENT_PATH not found, using %d as updated_at", prUpdatedAt))
```

Comment:
We can not fall back here, we will need the event.

Reply:
we discussed in a long call today about introducing a `TM_TEST_IGNORE_GITHUB_EVENT_FILE` (that's only available in a test build) for making it easy to locally test this and then fallback will happen only in this case.
We agree that a silent fallback here is not correct.

I think @wmalik will introduce this test env in the new PR.